app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
```

Comment:
Add blank line between different groups of imports in the future to follow PEP-8